tests/test_content_loader.py

```diff
@@ -0,0 +1,270 @@
+# coding=utf-8
+
+import unittest
+import mock
+
+import os
+import tempfile
+import yaml
+
+from dmutils.content_loader import ContentLoader
+
+
+class MonkeyPatch:
+    def __init__(self, mocked_content=""):
+        self.mocked_content = mocked_content
+
+    def read_yaml_file(self, yaml_file):
+        return yaml.load(self.mocked_content[yaml_file])
+
+    def yaml_file_exists(self, file):
+        return True
+
+
+class TestContentLoader(unittest.TestCase):
```

Comment:
Modifying `ContentLoader` here will keep the replaced methods after the test ends. I think the best way to do this is to use `mock.patch("dmutils.content_loader.ContentLoader._yaml_file_exists")` and `mock.patch("dmutils.content_loader.ContentLoader._read_yaml_file")` either in the `setUp`/`tearDown` methods or as class/method decorators